frontend/src/metabase/admin/datamodel/metadata/components/MetadataTable/MetadataTable.tsx

```diff
@@ -91,7 +96,7 @@ const MetadataTable = ({
 
   const handleChangeVisibility = useCallback(
     (visibility: TableVisibilityType) => {
-      onUpdateTable(table, "visibility_type", visibility);
+      onUpdateTable(table, "visibility property", visibility, true);
```

Comment:
_non-blocking comment_

While we're at it, this string should be translated:

```suggestion
      onUpdateTable(table, t`visibility property`, visibility, true);
```

We could also do the same in `handleChangeName` and `handleChangeDescription` above.